report/ApiTesting.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
please check again you are pushing reports
you can use the rebase to make this change a single commit after deleting the report, then a git push -f to rewrite your branch to never have this on its commits